src/main/index.js

```diff
@@ -14,21 +14,17 @@ function createMainWindow() {
     },
   })
 
-  if (isDevelopment) {
-    window.webContents.openDevTools()
-  }
+  isDevelopment && window.webContents.openDevTools();
```

Comment:
This doesn't look really necessary, can you at least add a comment explaining what it does?